superset-frontend/package.json

```diff
@@ -91,7 +91,7 @@
     "@superset-ui/legacy-preset-chart-big-number": "^0.17.42",
     "@superset-ui/legacy-preset-chart-deckgl": "^0.4.6",
     "@superset-ui/legacy-preset-chart-nvd3": "^0.17.42",
-    "@superset-ui/plugin-chart-echarts": "^0.17.44",
+    "@superset-ui/plugin-chart-echarts": "^0.17.47",
```

Comment:
I think bumping this to `0.17.47` without also bumping `superset-ui/core` first to `0.17.46` might be problematic. I suggest merging this first: #14547 , after which this should update cleanly.